tests/simplify.test.sh

```
@@ -340,6 +340,26 @@ assert_in_section "$CODE_FIXER" '^## Process' '^## Refusal triggers' \
   'any other source attribute is malformed' \
   "E4.3 — code-fixer Step 1 still refuses non-member sources (input-malformed)"
 
+echo
+echo "== E5: standalone deferred-findings handoff is Phase-2-only =="
+DEFER_LINE="$(grep '^DEFER_INPUTS=' "$SIMPLIFY" | head -n1)"
+AGG_PATH="/repo/.uberdev/research/20260726-010203-abcdef0/simplify-final.md"
```

Shellcheck (0.11.0)

[warning] AGG_PATH appears unused. Verify use (or export if used externally).

(SC2034)